assets/app/scripts/controllers/overview.js

```diff
@@ -69,6 +70,11 @@ angular.module('openshiftConsole')
       Logger.log("services (list)", $scope.services);
     }));
 
+    watches.push(DataService.watch("routes", $scope, function(routes) {
+      $scope.routes = routes.by("serviceName");
```

Comment:
Infinity.  The zero case and the first route with a host, then first route case are relevant.  Maybe a little # in the corner listing number of routes

> On May 7, 2015, at 10:46 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In assets/app/scripts/controllers/overview.js:
> 
> > @@ -69,6 +70,11 @@ angular.module('openshiftConsole')
> >        Logger.log("services (list)", $scope.services);
> >      }));
> >  
> > +    watches.push(DataService.watch("routes", $scope, function(routes) {
> > +      $scope.routes = routes.by("serviceName");
> can you have more than one route for a service?
> 
> —
> Reply to this email directly or view it on GitHub.
>